server/users/index.js

```diff
@@ -43,14 +43,14 @@ router.get('/:userId', async (req, res, next) => {
   }
 });
 
-router.get('/:username/usermeta', async (req, res, next) => {
+router.getUsermetaMiddleware = (helperP = helpers, metaList = process.env.USER_METADATA_LIST) => async (req, res, next) => {
 	try {
-		helpers.dynamicMetadata(req.params.username, process.env.USER_METADATA_LIST, (err, metaInfos) => {
+		helperP.dynamicMetadata(req.params.username, metaList, (err, metaInfos) => {
 			if (err)
 				return next(err);
 
 			let result = [];
-      let allkeys = process.env.USER_METADATA_LIST.split(',');
```

Comment:
I realize I didn't send the changes we made in this files while in Zoom with you. Sorry you had to redo it.

There are indentation issues here and below, makes the code and diff less readable. Don't fix them for now, I'll do it automatically once everything is merged in develop.

Reply:
Its ok regarding the changes .. 
But the thing that the indentation looks fine in vs, but dunno why here it's showing as good as i see it.
I used the [https://beautifier.io/](url) to do the indentation...